### PR TITLE
[WIP] Exporting plants that have omnizine or earthsblood will give supply points

### PR DESCRIPTION
**Pull request by ma44**
Cause exporting rare healthy medicine is good

Also I'm adding a cap on how many points the supply points can get up to, so it won't easily make hundreds of thousands of supply points in several minutes and mass order NULLs without exporting other things. 

TODO: Test and compile
tweak numbers for balance

### CODE REVIEW

File: code/modules/cargo/exports/plantmed.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/datum/export/plantmed/get_cost(obj/O)
+        var/obj/item/seeds/S = O
+        if S.has_reagent("omnizine, 1)
+                S.reagents.get_reagent_amount("omnizine")
+                        return ..() * cost
+                
+        if S.has_reagent("earthsblood, 1)
+                S.reagents.get_reagent_amount("earthsblood")
+                        return ..() * cost
+
```
Comment (nullbear) [inline code/modules/cargo/exports/plantmed.dm]: Er, don't you mean 

` return S.reagents.get_reagent_amount("earthsblood") * cost` 

??

File: code/modules/cargo/exports/plantmed.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                        return ..() * cost
+                
+        if S.has_reagent("earthsblood, 1)
+                S.reagents.get_reagent_amount("earthsblood")
+                        return ..() * cost
+                
+/datum/export/plantmed/sell_object(obj/O)
+        var/cost = ..()
+        if SSshuttle.points = 14000
+                return 0
```
Comment (nullbear) [inline code/modules/cargo/exports/plantmed.dm]: Not really sure what this does. But this isn't a proper if statement.

should probably be:

if (SSshuttle.points >= 14000)
    return 0

As  a single = sign SETS a variable. Two == compares it. And comparing to see if the shuttle points are exactly 14000 is a really edge case.

File: code/modules/cargo/exports/plantmed.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/datum/export/plantmedicine
+        cost = 2 //Gets multiplied based on healing chems inside of seed
+        unit_name = Medicine
+        export_types = list(/obj/item/seeds)
+        SSshuttle.points = SSshuttle.points
+
```
Comment (nullbear) [inline code/modules/cargo/exports/plantmed.dm]: SSshuttle.points = SSshuttle.points does absolutely nothing.

File: code/modules/cargo/exports/plantmed.dm [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+/datum/export/plantmed/get_cost(obj/O)
+        var/obj/item/seeds/S = O
+        if S.has_reagent("omnizine, 1)
+                S.reagents.get_reagent_amount("omnizine")
+                        return ..() * cost
```
Comment (nullbear) [inline code/modules/cargo/exports/plantmed.dm]: Seeds do not contain reagents. Only the grown/plant does.

**Comment by Jalleo** [comment]
I wonder what @Cheridan thinks.

**Comment by ma44** [comment]
"PR has been closed because of jobban from botany"

**Comment by Cheridan** [comment]
Stop this shit. Don't try and re-add exploits that we've removed because of your own abuse.

**Pull request closed without merging**